--- web2ldap-0.15.2/etc/web2ldap/web2ldapcnf/standalone.py	2004-10-31 12:07:22.000000000 +0100
+++ web2ldap-0.15.2/etc/web2ldap/web2ldapcnf/standalone.py.fix	2004-10-31 12:09:32.000000000 +0100
@@ -22,7 +22,8 @@
 access_allowed = ['127.0.0.0/255.0.0.0']
 
 # User account name to setuid after being started as root
-run_username = 'web2ldap'
+#run_username = 'web2ldaprun'
+run_username = 'http'
 
 # Contact mail address of the server admin when running stand-alone
 server_admin = 'admin@web2ldap.de'
