--- web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/__init__.py.fix	2003-04-10 09:38:52.000000000 +0200
+++ web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/__init__.py	2003-08-02 23:42:24.000000000 +0200
@@ -11,19 +11,22 @@
 import sys,os
 
 # Directory where web2ldap is started
-web2ldap_dir = os.path.dirname(os.path.dirname(os.path.abspath(sys.argv[0])))
+web2ldap_dir = '/usr/share/web2ldap'
 
 # Default directory for [web2ldap]/var
-var_dir = os.path.join(web2ldap_dir,'var')
-#var_dir = '/var'
+#var_dir = os.path.join(web2ldap_dir,'var')
+var_dir = '/var'
 
 # Default directory for [web2ldap]/var
-etc_dir = os.path.join(web2ldap_dir,'etc')
-#etc_dir = '/etc'
+#etc_dir = os.path.join(web2ldap_dir,'etc')
+etc_dir = '/etc'
 
 # Default directory for [web2ldap]/var
 templates_dir = os.path.join(etc_dir,os.path.join('web2ldap','templates'))
 
 # Several default sub-directories in [web2ldap]/var
-for var_subdir in ['run','log','state']:
-  vars()['var_'+var_subdir] = os.path.join(var_dir,var_subdir)
+#for var_subdir in ['run','log','state']:
+#  vars()['var_'+var_subdir] = os.path.join(var_dir,var_subdir)
+var_run='/var/run/web2ldap'
+var_log='/var/log/web2ldap'
+var_state='/var/lib/web2ldap'
--- web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/fastcgi.py.fix	2003-08-02 23:42:50.000000000 +0200
+++ web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/fastcgi.py	2003-08-02 23:44:04.000000000 +0200
@@ -11,7 +11,8 @@
 run_threaded = 1
 
 # Path name of PID file
-pid_file = '/tmp/web2ldap-fastcgi.pid'
+#pid_file = '/tmp/web2ldap-fastcgi.pid'
+pid_file = '/var/run/web2ldap/web2ldap-fastcgi.pid'
 
 # Path name of error log file
 # error_log = '/var/log/web2ldap/error_log'
--- web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/scgi.py.fix	2003-08-02 23:44:17.000000000 +0200
+++ web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/scgi.py	2003-08-02 23:45:16.000000000 +0200
@@ -12,4 +12,5 @@
 prefix = '/web2ldap-scgi/web2ldap.py'
 
 # Path name of PID file
-pid_file = '/tmp/web2ldap-scgi.pid'
+#pid_file = '/tmp/web2ldap-scgi.pid'
+pid_file = '/var/run/web2ldap/web2ldap-scgi.pid'
--- web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/standalone.py.fix	2003-08-02 23:45:34.000000000 +0200
+++ web2ldap-0.11.24/etc/web2ldap/web2ldapcnf/standalone.py	2003-08-02 23:46:28.000000000 +0200
@@ -22,7 +22,8 @@
 access_allowed = ['127.0.0.0/255.0.0.0']
 
 # User account name to setuid after being started as root
-run_username = 'web2ldaprun'
+#run_username = 'web2ldaprun'
+run_username = 'http'
 
 # Contact mail address of the server admin when running stand-alone
 server_admin = 'admin@web2ldap.de'
